OpenKeychain/src/main/java/org/sufficientlysecure/keychain/operations/ImportExportOperation.java

```diff
@@ -376,8 +385,6 @@ public ImportKeyResult importKeyRings(Iterator<ParcelableKeyRing> entries, int n
             log.add(LogType.MSG_IMPORT_ERROR, 1);
         }
 
-        ContactSyncAdapterService.requestSync();
```

Comment:
I thought it would be better to sync with contacts at the end of a batch of imports, especially when it comes to multi-threaded, rather than sync after each key. I've mentioned it at the beginning of the function and at line 343.